R/read.R

```diff
@@ -1,11 +1,12 @@
 orderly_read <- function(path, call = NULL) {
-  assert_file_exists_relative("orderly.R", name = "Orderly file",
+  orderly_name <- deprecate_old_orderly_name(path, basename(path))
+  assert_file_exists_relative(orderly_name, name = "Orderly file",
                               workdir = path, call = call)
-  orderly_read_r(file.path(path, "orderly.R"))
+  orderly_read_r(file.path(path, orderly_name), orderly_name)
 }
 
 
-orderly_read_r <- function(path) {
+orderly_read_r <- function(path, orderly_name) {
```

Comment:
```suggestion
orderly_read_r <- function(path) {
```

Argument isn't being used anywhere.

Reply:
this is being used for ret$resources line in the function towards the end of it

Reply:
Yeah, missed that sorry